Testscripts/Linux/VERIFY-LINUX-CONFIGURATION.py

```diff
@@ -99,14 +100,27 @@ def VerifyBashHistory():
         RunLog.info("Content:\n%s" % hist_file_content)
 
 
+def VerifyIrqbalanceExist():
+    global irqbalance_verify_result
+    result = Run("(command -v rpm > /dev/null && rpm -qa | grep -i irqbalance | wc -l) || (command -v dpkg > /dev/null && dpkg -l | grep -i irqbalance | wc -l)")
```

Comment:
Can we simply this by just checking irqbalance utility is present or not.
```command -v irqbalance > /dev/null```

Reply:
good point, let me double check.